packages/cli/src/cli/commands/init.test.ts

```diff
@@ -0,0 +1,128 @@
+import fs, { readFileSync } from "fs";
+import path from "path";
+import tmp from "tmp";
+import { sh } from "./init";
+
+jest.setTimeout(180000);
+
+const PCC = "./dist/index.js";
+
+beforeAll(async () => {
+  await sh("pnpm run build:staging");
+});
+
+test("should be able to init starter kit for nextjs template", async () => {
+  const appFolder = tmp.tmpNameSync();
+  await sh(`${PCC} init ${appFolder} --template nextjs --use-pnpm`);
+
+  // Dependencies are installed
+  expect(fs.existsSync(`${appFolder}/node_modules`)).toBe(true);
+  expect(fs.existsSync(`${appFolder}/pnpm-lock.yaml`)).toBe(true);
+
+  // Eslint not initialized
+  // expect(fs.existsSync(`${appFolder}/.eslintrc.json`)).toBe(false);
```

Comment:
Can we remove this if it's not needed?

Reply:
I have pushed one change in the starter kit regarding `.eslintrc.json` in the same PR.
Once it's pushed to the GitHub repo, the commented assert will start working.  
I will raise one more PR to 
- Uncomment above assert
- Remove `CI=false` from test workflow